Makefile

```diff
@@ -110,7 +110,7 @@ $(OBJ_EXAMPLE): $(EXAMPLE)/$(EXAMPLE).c $(DEPS)
 	@mkdir -p $(@D)
 	$(CC) $(MMDOPT) -c -o $@ $< $(CFLAGS) -Iinclude
 
-$(OBJ_TESTS): $(TESTS)/$(TESTS).c $(DEPS)
```

Comment:
I could have sworn it was working so that's why I explicitly added it, but I must be hallucinating, since it's clearly done automatically 
![IMG_7937](https://github.com/aremmell/libsir/assets/61629094/df3dc4ca-cfc8-48e1-9e9f-8065c76f3413)
I'll fix